packages/account_sdk/src/controller.rs

```diff
@@ -65,6 +66,23 @@ where
         address: Felt,
         chain_id: Felt,
         backend: B,
+    ) -> Self {
+        Self::with_guardian(
+            app_id, username, class_hash, provider, owner, None, address, chain_id, backend,
+        )
+    }
+
+    #[allow(clippy::too_many_arguments)]
+    pub fn with_guardian(
+        app_id: String,
+        username: String,
+        class_hash: Felt,
+        provider: P,
+        owner: Signer,
+        guardian: Option<Signer>,
```

Comment:
im not sure if we want this here. since the sdk would never be it's own guardian. the guardian signer would come from the cartridge backend